pyproject.toml

```diff
@@ -77,11 +77,10 @@ dev = [
   "twine",       # needed to publish to pypi
 ]
 test = [
-  "polyfactory>=2", # factory for pydantic models
-  "pytest>=2.7.3",  # testing framework
-  "pytest-cov",     # test coverage
-  "pytest-mock",    # mocking
-  "pytest-timeout", # darn hanging tests
+  "pytest>=2.7.3",       # testing framework
```

Comment:
is polyfactory no longer necessary?

Reply:
Yeah I disliked how it works, and it also broke quickly when customizing some pydantic classes. With some recent commentary from @deltamarnix about dependency explosion I decided to take it out.

Reply:
nice.